### ISSUE TITLE
Loss scaling params in SOC training

### ISSUE THREAD

**Issue by scrimm-hash** [ZHKKKe/MODNet#61]
Hello.
Thanks for sharing training code.
There's the following note in `soc_adaptation_iter` function about `soc_semantic_scale` and `soc_detail_scale` params: NOTE: please adjust according to your dataset.

Could you please share the logic for choosing these values?

**Comment by ZHKKKe** [comment]
@scrimm-hash 
Hi. The default values should work well in most cases. You only need to adjust them when the losses are too large or too small.
